### PR TITLE
Feature/create an excercice page

### PR DESCRIPTION
**Pull request by EthannSchneider**

### CODE REVIEW

File: src/models/databases_connectors/databases_access.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+interface DatabasesAccess
+{
+	public function isExcerciceExist(int $id): bool;
```
Comment (Guillaume1868) [inline src/models/databases_connectors/databases_access.php]: doesExerciceExist would make more sense

File: src/models/databases_connectors/databases_access.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+	public function createExercice(string $title): int;
+
+	public function getExcerciceTitleFromId(int $id): string;
```
Comment (Guillaume1868) [inline src/models/databases_connectors/databases_access.php]: Why make the function name to precise the parameter, when the parameter has a clear name

change it fot getExerciceTitle(int $id)

In future cases we could add function overload with other parameter type

File: src/models/databases_connectors/postgresql/postgresql_access.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	{
+		return count($this->postgresql->select('SELECT id FROM excercices WHERE id = :id', [':id' => $id])) > 0;
+	}
+
+	public function createExercice(string $title): int
+	{
+		return (int)$this->postgresql->select('INSERT INTO excercices (title) VALUES (:title) RETURNING id', [':title' => $title])[0][0];
+	}
+
+	public function getExcerciceTitleFromId(int $id): string
```
Comment (Guillaume1868) [inline src/models/databases_connectors/postgresql/postgresql_access.php]: Same as above

File: src/models/databases_connectors/postgresql/postgresql_access.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+{
+	private $postgresql;
+
+	public function __construct($host, $port, $dbname, $postgres_user, $postgres_password)
+	{
+		$this->postgresql = new Postgresql($host, $port, $dbname, $postgres_user, $postgres_password);
+		$this->create_db_if_not_exist();
+	}
+
+	public function isExcerciceExist(int $id): bool
```
Comment (Guillaume1868) [inline src/models/databases_connectors/postgresql/postgresql_access.php]: same as above

**Review by Guillaume1868 — CHANGES_REQUESTED** [review]
Good work, please just change the wording details

File: src/models/databases_connectors/databases_access.php (hunk 1 above)
Comment (EthannSchneider) [reply src/models/databases_connectors/databases_access.php]: "Does" is used to describe actions and is the third person singular form of the verb "do". "Is" describes the condition or existence of a subject and is the third person singular form of the verb "be".

**Review by Guillaume1868 — APPROVED** [review]
lgtm

**Pull request merged**